test/testdata/deployednettemplates/recipes/fastnet/node.json

```diff
@@ -0,0 +1,10 @@
+{
+    "APIEndpoint": "{{APIEndpoint}}",
+    "APIToken": "{{APIToken}}",
+    "EnableBlockStats": true,
+    "EnableTelemetry": true,
+    "TelemetryURI": "{{TelemetryURI}}",
+    "EnableMetrics": true,
+    "MetricsURI": "{{MetricsURI}}",
+    "ConfigJSONOverride": "{ \"TxPoolExponentialIncreaseFactor\": 1, \"DNSBootstrapID\": \"<network>.algodev.network\", \"DeadlockDetection\": -1, \"PeerPingPeriodSeconds\": 30, \"EnableAgreementReporting\": true, \"EnableAgreementTimeMetrics\": true, \"EnableAssembleStats\": true, \"EnableProcessBlockStats\": true, \"BaseLoggerDebugLevel\": 4, \"EnableProfiler\": true, \"CadaverSizeTarget\": 0, \"EnableAccountUpdatesStats\": true}"
```

Comment:
i think EnableRuntimeMetrics should appear here if you generate this file using a version of netgoal from go-algorand since #4041 was merged.

Reply:
Good thinking.